raiden/ui/runners.py

```diff
@@ -62,17 +59,17 @@ def run(self):
             log.debug("Using config file", config_file=self._options["config_file"])
 
     def _start_services(self) -> None:
-        config = deepcopy(App.DEFAULT_CONFIG)
-        config["reveal_timeout"] = self._options["default_reveal_timeout"]
-        config["settle_timeout"] = self._options["default_settle_timeout"]
-        if self._options.get("extra_config", dict()):
-            merge_dict(config, self._options["extra_config"])
-            del self._options["extra_config"]
-        self._options["config"] = config
+        # config = deepcopy(App.DEFAULT_CONFIG)
```

Comment:
Why is this code commented out? I guess WIP? (Just realized PR is still a draft)